cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -134,6 +145,15 @@ public static void pushPackages(String packageName, String sourceRoot, String in
             if (!installToRepo.equals("home")) {
                 throw new BLangCompilerException("Unknown repository provided to push the package");
             }
+            if (!pushToRepo) {
+                outStream.print("Install the artifact " + orgName + "/" + packageName + ":" + version  + " to the " +
+                                        "home repository [yes/y, no/n]: (y) ");
+                Scanner scanner = new Scanner(System.in, Charset.defaultCharset().name());
+                String buildOnPush = scanner.nextLine().trim();
+                if (buildOnPush.equalsIgnoreCase("no") || buildOnPush.equalsIgnoreCase("n")) {
```

Comment:
Same here. If some one say anything other than no/n , this will get installed right? If yes, shall we check for the yes/y and proceed?

Reply:
Added the check with 3b98edc